Resources/Prototypes/Entities/Clothing/OuterClothing/sinditrench.yaml

```
@@ -0,0 +1,20 @@
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: Republicssinditrench
+  name: офицерский тренч синдиката
+  description: Знаменитый тренч офицеров Синдиката. Не многие переживали встречу с их владельцами.
```

```suggestion
  description: Знаменитый тренч офицеров Синдиката. Не многие переживали встречу с их владельцами.
```
Надо через локализацию